picachu/tests/auth/test_auth.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from flask import url_for
+
+from picachu.config.auth_config import auth_username, auth_password
+
+
+def test_log_in_happy_path(flask_app):
+    data = {
+        'login': auth_username,
+        'password': auth_password,
+    }
+
+    response = flask_app.post(url_for('api.auth.log_in'), json=data)
+
+    assert response.status_code == 202
```

Comment:
We need to check not only the status code, but also the scheme from which the answer comes. It is important to check that the right contract is returned.